src/kvazaar.h

```diff
@@ -34,6 +34,10 @@
 extern "C" {
 #endif
 
+#if defined(KVZ_STATIC_LIB) && defined(PIC)
+  #undef KVZ_STATIC_LIB
```

Comment:
This was a dirty workaround to pass separate flags for shared and static libraries, which is otherwise a bit difficult using libtool and automake. Maybe libtool should only be used to generate shared libraries and I should add a separate build target for the static library?